RedBugs_webapp/src/main/java/com/isa/pl/redbugs/controller/RouteController.java

```diff
@@ -17,6 +25,44 @@ public RouteController(RouteService routeService) {
     @GetMapping("/routes")
     public String getRoutes() throws IOException {
         routeService.findAllRoutes();
-        return "routes";
+        return "vehicle_form";
     }
+
+    @GetMapping("/routes/details/{routeId}")
+    public String getRouteDetailsById(@PathVariable("routeId") Long routeId, Model model) throws IOException {
+        Route routeFound = routeService.findRouteById(routeId);
+        model.addAttribute("route", routeFound);
+
+        String pageTitle = "List of stops bus number " + routeId;
+        model.addAttribute("pageTattle", pageTitle);
+
+        List<String> stopList = routeService.findAllStopsOnRoute(routeId);
+        model.addAttribute("stops", stopList);
+        return "details_of_route";
+    }
+
+    @GetMapping("/routes/edit/{routeId}")
+    public String getEditRouteById(@PathVariable("routeId") Long routeId, Model model) throws IOException {
+        Route routeFound = routeService.findRouteById(routeId);
+        model.addAttribute("route", routeFound);
+
+        String pageTitle = "Editing route number: " + routeId;
```

Comment:
Jeśli decydowalibyśmy się na polski, to "Edycja trasy numer: "

Reply:
Dziękuję. Poprawiłem